.github/workflows/translations-sync.yml

```diff
@@ -109,27 +107,19 @@ jobs:
       - name: Install packages
         run: pnpm install --frozen-lockfile
 
-      - name: Run `lint:md --fix`
-        # This runs a specific version of ESLint with only the Translation Pages Globbing
-        # This avoid that unrelated changes get linted/modified within this PR
-        run: pnpm exec eslint "apps/site/pages/**/*.md?(x)" --fix --cache --cache-strategy=metadata --cache-file=apps/site/.eslintmdcache --config=apps/site/eslint.config.js
+      # Re-stage latest commit and run lint-staged
+      - name: Re-stage and Format Files
+        run: |
+          # Soft reset to unstage the commit but keep changes
+          git reset --soft HEAD^
 
-      - name: Run `prettier --write`
-        # This runs a specific version of Prettier with only the Translation Pages Globbing
-        # This avoid that unrelated changes get prettied/modified within this PR
-        run: pnpm exec prettier "apps/site/{pages,i18n}/**/*.{json,md,mdx}" --check --write --cache --cache-strategy=metadata --cache-location=apps/site/.prettiercache
+          # Run lint-staged on the staged files
+          # This will run linters/formatters only on changed files
+          # according to the configuration in package.json or .lintstagedrc
+          pnpm lint-staged
```

Comment:
overkill using this, albeit cleaner and not without drift

https://github.com/nodejs/nodejs.org/blob/main/.husky/pre-commit

we dont need typechecking or the whole project linted https://github.com/nodejs/nodejs.org/blob/main/.lintstagedrc.json as crowdin is only changing a subset. (I know it only should work on changed files)

can we call https://github.com/nodejs/nodejs.org/blob/main/package.json#L25 instead?